sdk/go/x/auto/minimum_version.go

```diff
@@ -18,6 +18,6 @@ import "github.com/blang/semver"
 
 var minimumVersion = semver.Version{
 	Major: 2,
-	Minor: 21,
+	Minor: 25,
 	Patch: 0,
 }
```

Comment:
We'll need this to include "-alpha.0" otherwise our dev builds break.

Reply:
Or if we want to be specific it should be `-alpha.[whatever-build-this-will-be-once-merged]` but the latest build on npmsjs is `alpha.1618004182`

Reply:
Applied and tested this locally on go and node with my current dev build.